docs/Customizing Look and Functionality/Customizing Help Prompts/index.md

```diff
@@ -181,6 +206,18 @@ sdlManager.sendRPC(setGlobalProperties);
 ```
 !@
 
+@![javascript]
+```js
+const setGlobalProperties = new SDL.rpc.messages.SetGlobalProperties();
+const chunk = new SDL.rpc.structs.TTSChunk().setText('Your custom help prompt').setType(SDL.rpc.enums.SpeechCapabilities.TEXT);
+setGlobalProperties.setTimeoutPrompt([chunk]);
+const response = await sdlManager.sendRpc(setGlobalProperties);
+if (response instanceof SDL.rpc.RpcResponse) {
+    // The help prompt is updated
```

Comment:
This and the above really should match the iOS / Java snippets with telling devs how to handle errors and such.